docs/source/topics/ref_minimum-system-requirements.adoc

```diff
@@ -50,7 +50,7 @@ To assign more resources to the {prod} instance, see link:{crc-gsg-url}#configur
 
 === Linux
 
-* On Linux, {prod} is supported only on {rhel}/{centos} 7.5 or later (including 8.x versions) and on the latest two stable {fed} releases.
+* On Linux, {prod} is supported only on the latest two {rhel}/{centos} 7 and 8 minor releases and on the latest two stable {fed} releases.
```

Comment:
Doesn't latest is now RHEL-9 so it should be `8 and 9 minor releases` ?

Reply:
Ah yes it's released now, I was thinking it would be released later this year, and that we'd have to update that statement :)